### PR TITLE
Add icon for xfce4-terminal

### PR DESCRIPTION
**Pull request by fredden**

### CODE REVIEW

**Comment by justbuchanan** [comment]
Thanks for the PR! I have a few suggestions I'll make inline.

File: autoname_workspaces.py [hunk 1]
```diff
@@ -91,6 +92,9 @@ def rename_workspaces(i3):
     for ws_index, workspace in enumerate(i3.get_tree().workspaces()):
         ws_info = ws_infos[ws_index]
 
+        if ws_info != focused_workspace(i3):
+            continue
```
Comment (justbuchanan) [inline autoname_workspaces.py:96]: What happens if a window opens or closes on a workspace that isn't currently focused? For example, sometimes I have a software update program running on a hidden workspace that closes itself when done.

**Comment by fredden** [comment]
Odd, I'd intended this PR to be only adding the icon, and was going to do another for the other commits.

File: autoname_workspaces.py [hunk 2]
```diff
@@ -136,5 +144,9 @@ def window_event_handler(i3, e):
         if e.change in ['new', 'close', 'move']:
             rename_workspaces(i3)
 
+    def workspace_event_handler(i3, e):
+        rename_workspaces(i3)
```
Comment (justbuchanan) [inline autoname_workspaces.py:148]: I think we only want to trigger a rename when a workspace is created or deleted. Similar to the `window_event_handler()`, we should check the value of `e.change`.

See the docs here for what values "change" could be https://i3wm.org/docs/ipc.html#_events.

**Review by justbuchanan — CHANGES_REQUESTED** [review]

File: autoname_workspaces.py (hunk 2 above)
Comment (justbuchanan) [reply autoname_workspaces.py:148]: You can also try this experimentally by saving this chunk of code into a python file and running it:

~~~{.py}
import i3ipc

def handle(i3, e):
    print(e)
    print(e.change)

i3 = i3ipc.Connection()
i3.on('workspace', handle)
i3.main()
~~~

**Comment by fredden** [comment]
Thanks for the comments. I'll close this and submit only what I intended to be included in this pull request, and we can try again.

**Pull request closed without merging**